src/reference/00-Getting-Started/02-sbt-by-example.md

```diff
@@ -167,13 +165,23 @@ ThisBuild / scalaVersion := "2.12.6"
 
 ### Name your project
 
-Using an editor, change `build.sbt` to follows:
+Using an editor, change `build.sbt` as follows:
 
 @@snip [name]($root$/src/sbt-test/ref/example-name/build.sbt) {}
 
+These lines set the `organization` and `scalaVersion` values
+for your entire build, and define one project within the build
+that has a given `name`; the `organization` and `scalaVersion` used
+by this project are taken from the values defined for the build.
```

Comment:
The point of this page is just to take them through the experience of sbt without too much explanation.
So I'd cut these.